### PR TITLE
Create legesher.yml

### PR DESCRIPTION
**Pull request by Beomus**

### CODE REVIEW

File: legesher.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+name: Up-For-Grabs.net
+desc: The bridge between the words we speak and the code we write.
+site: https://github.com/legesher/legesher
+tags:
+- html/css
+- ruby
+- python
+- javascript
+upforgrabs:
+  name: help needed with various projects
```
Comment (shiftkey) [inline legesher.yml]: This field needs to match with the label used in the project:

```suggestion
  name: Good First Issue
```

File: legesher.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: Up-For-Grabs.net
+desc: The bridge between the words we speak and the code we write.
+site: https://github.com/legesher/legesher
+tags:
+- html/css
```
Comment (shiftkey) [inline legesher.yml]: This value should be split into two separate values:

```suggestion
- html
- css
```

**Review by shiftkey — CHANGES_REQUESTED** [review]
@nemequ thanks for submitting!

Please rename `legesher.yml` to `_data/projects/legesher.yml` so that it will be listed on the site, and there's also some suggested changes I have to get this ready to merge.

**Comment by Beomus** [comment]
Thank you for your review! I have made changes according to your suggestions, please let me know if you need me to change anything else.

File: _data/projects/legesher.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+site: https://github.com/legesher/legesher
+tags:
+- html
+- css
+- ruby
+- python
+- javascript
+upforgrabs:
+  name: Good First Issue
+  link: https://github.com/legesher/legesher/labels/Good%20First%20Issue
```
Comment (shiftkey) [inline _data/projects/legesher.yml]:
```suggestion
  link: https://github.com/legesher/legesher/labels/good%20first%20issue
```

File: _data/projects/legesher.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+desc: The bridge between the words we speak and the code we write.
+site: https://github.com/legesher/legesher
+tags:
+- html
+- css
+- ruby
+- python
+- javascript
+upforgrabs:
+  name: Good First Issue
```
Comment (shiftkey) [inline _data/projects/legesher.yml]:
```suggestion
  name: good first issue
```

**Review by shiftkey — CHANGES_REQUESTED** [review]
Very close -  just ensuring the label name matches what's in use

**Review by shiftkey — APPROVED** [review]

**Comment by shiftkey** [comment]
Thanks @Beomus!

**Pull request merged**